src/components/entity/ha-state-label-badge.ts

```diff
@@ -166,6 +168,31 @@ export class HaStateLabelBadge extends LitElement {
     }
   }
 
+  private _computeUpperLabel(
+    domain: string,
+    state: HassEntity
+  ): string | undefined {
+    if (["device_tracker"].includes(domain) && state.attributes.battery) {
+      return state.attributes.battery + "%";
+    }
+
+    return undefined;
+  }
+
+  private _computeUpperLabelIcon(
+    domain: string,
+    state: HassEntity
+  ): string | undefined {
+    if (["device_tracker"].includes(domain) && state.attributes.battery) {
+      const battery = Math.round(state.attributes.battery / 10) * 10;
```

Comment:
Does rounding the attr not work here? why divide and then multiply

Reply:
I want to round to the nearest ten for the icon.  Probably won't keep this around, I don't think the icon adds much